src/main/kotlin/commands/Nick.kt

```diff
@@ -89,10 +89,17 @@ class Nick(private val chattORE: ChattORE) : BaseCommand() {
 
     @Subcommand("presets")
     @CommandPermission("chattore.nick.preset")
-    @CommandCompletion("@username")
-    fun presets(player: Player, @Optional shownText: String?) {
+    @CommandCompletion("pride|birdflop|players")
+    fun presets(player: Player, @Optional shownText: String?, @Optional type: String?) {
+        val presets = when (type?.lowercase()) {
+            "pride" -> pridePresets
```

Comment:
It is probably better to separate each of these types of presets into separate configuration items. It may even be better to make the presets a map of keys (preset category) associated with a map itself of keys (preset name) and the preset value itself. This would allow us to define more types of presets later.

Reply:
Yea I like the map of maps system with the nickname_presets [all presets combined] > categorys [e. g. pride] > preset [e.g. nickster]